go.mod

```diff
@@ -14,8 +14,8 @@ require (
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.17.0
 	github.com/operator-framework/api v0.14.1-0.20220413143725-33310d6154f3
-	github.com/operator-framework/helm-operator-plugins v0.0.10
-	github.com/operator-framework/java-operator-plugins v0.3.0
+	github.com/operator-framework/helm-operator-plugins v0.0.8
+	github.com/operator-framework/java-operator-plugins v0.1.0
```

Comment:
@varshaprasad96 @ryantking  
Do we need to update these plugins first now before updating SDK?

Reply:
See: https://github.com/operator-framework/helm-operator-plugins/pull/169

Reply:
solved